packages/components/addon/components/hds/gutters/index.js

```diff
@@ -0,0 +1,99 @@
+import Component from '@glimmer/component';
+import { htmlSafe } from '@ember/template';
+import { assert } from '@ember/debug';
+
+export const DIRECTIONS = ['horizontal', 'vertical'];
+export const SPACING_SIZES = ['xxs', 'xs', 'sm', 'md', 'lg', 'xl', 'xxl'];
+
+const regEx = /^((\d+)|(\d+\.\d+)|(\.\d+))(px|rem|em|%)$/;
```

Comment:
If we use the `style` Ember helper, it automatically validates the CSS unit for us (if the unit is not correct, it simply ignores the declaration is my understanding).

Reason: we avoid making a choice between which CSS units to support and which to leave out.

Reply:
I'll work on updating my code, thanks